independent-projects/tools/devtools-common/pom.xml

```diff
@@ -106,6 +106,12 @@
             <groupId>jakarta.websocket</groupId>
             <artifactId>jakarta.websocket-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.immutables</groupId>
+            <artifactId>value</artifactId>
+            <version>2.8.8</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Just to clarify, this dependency will be provided at runtime by something? What will it be?

Reply:
No, this is only used to generate sources. The annotations in this dependency don't exist at runtime

Reply:
thanks